pro/src/core/Offers/adapters/updateIndividualOffer/serializers.ts

```diff
@@ -39,37 +41,59 @@ export const serializeDurationMinutes = (
 
   return minutes + hours * 60
 }
-
-export const serializePatchOffer = (
+interface ISerializePatchOffer {
+  offer: IOfferIndividual
   formValues: Partial<IOfferIndividualFormValues>
-): PatchOfferBodyModel => ({
-  audioDisabilityCompliant:
-    formValues.accessibility &&
-    formValues.accessibility[AccessiblityEnum.AUDIO],
-  description: formValues.description,
-  extraData: serializeExtraData(formValues),
-  isNational: formValues.isNational,
-  isDuo: !!formValues.isDuo,
-  mentalDisabilityCompliant:
-    formValues.accessibility &&
-    formValues.accessibility[AccessiblityEnum.MENTAL],
-  motorDisabilityCompliant:
-    formValues.accessibility &&
-    formValues.accessibility[AccessiblityEnum.MOTOR],
-  name: formValues.name,
-  withdrawalDelay:
-    formValues.withdrawalDelay === undefined
-      ? null
-      : formValues.withdrawalDelay,
-  withdrawalDetails: formValues.withdrawalDetails || null,
-  visualDisabilityCompliant:
-    formValues.accessibility &&
-    formValues.accessibility[AccessiblityEnum.VISUAL],
-  withdrawalType: formValues.withdrawalType,
-  durationMinutes: serializeDurationMinutes(formValues.durationMinutes || ''),
-  bookingEmail: formValues.receiveNotificationEmails
-    ? formValues.bookingEmail
-    : null,
-  externalTicketOfficeUrl: formValues.externalTicketOfficeUrl || undefined,
-  url: formValues.url || undefined,
-})
+}
+
+export const serializePatchOffer = ({
+  offer,
+  formValues,
+}: ISerializePatchOffer): PatchOfferBodyModel => {
+  let sentValues: Partial<IOfferIndividualFormValues> = formValues
+  if (offer?.lastProvider) {
+    const {
+      ALLOCINE: allocineEditableFields,
+      ALL_PROVIDERS: allProvidersEditableFields,
+    } = SYNCHRONIZED_OFFER_EDITABLE_FIELDS
+
+    const asArray = Object.entries(formValues)
+    const editableFields = isAllocineOffer(offer)
+      ? [...allocineEditableFields, ...allProvidersEditableFields]
+      : allProvidersEditableFields
```

Comment:
(détail) je pense que c'est plus lisible
```suggestion
    const editableFields = allProvidersEditableFields
    if (isAllocineOffer(offer)) {
      editableFields.push(... allocineEditableFields)
    }
```